aiocouch/couchdb.py

```diff
@@ -39,3 +39,6 @@ def __init__(self, *args, **kwargs):
 
     async def keys(self, **params):
         return await self._server._all_dbs(**params)
+
+    async def info(self):
+        return await self._server._get('/')
```

Comment:
Please add an `_info()` method to RemoteServer and add the `@raises` decorator for the 401 to that new method. Just like the `_check_session` method there.

Reply:
Oh, and please use `"` instead of `'`. That's probably, what the black CI step is complaining about.

Reply:
Sure!